front/app/containers/ProjectsShowPage/shared/header/ProjectSharingModal.tsx

```diff
@@ -63,7 +56,15 @@ const ProjectSharingModal = memo<Props & InjectedIntlProps>(
           closeOnClickOutside={true}
           header={<T value={project.attributes.title_multiloc} />}
         >
-          <Container className={className}>
+          <Box
+            width="100%"
+            maxWidth="400px"
+            padding="40px 25px"
+            ml="auto"
+            mr="auto"
```

Comment:
You can do my instead of these two.

Reply:
I didn't think about it before, by "my" would impact top and bottom margins (not left and right). Looks like there is no "mh" for horizontal margins in the component library. Should I add this?

Reply:
Ah, of course, it's `mx` 🤦‍♀️ My bad!

Reply:
Ahh gotchya! I'll fix it then. Thanks!